tests/app/views/contexts/test_thank_you_context.py

```diff
@@ -52,12 +54,62 @@ def test_default_survey_context(fake_session_data, app: Flask):
         assert len(context["metadata"]["itemsList"]) == 2
 
 
+def test_view_submitted_response_enabled_is_true_when_set_in_schema(
+    fake_session_data, app: Flask
+):
+    with app.app_context():
+        context = build_thank_you_context(
+            fake_session_data, SUBMITTED_AT, SURVEY_TYPE_DEFAULT, SCHEMA
+        )
+
+        assert context["view_submitted_response"]["enabled"] is True
+
+
+def test_view_submitted_response_enabled_is_false_when_not_enabled_in_schema(
+    fake_session_data, app: Flask
+):
+    with app.app_context():
+        schema = load_schema_from_name("test_title", "en")
+
+        context = build_thank_you_context(
+            fake_session_data, SUBMITTED_AT, SURVEY_TYPE_DEFAULT, schema
+        )
+
+        assert context["view_submitted_response"]["enabled"] is False
+
+
+def test_view_submitted_response_expired_is_false_when_submitted_at_less_than_expiry_time(
+    fake_session_data, app: Flask
+):
+    with app.app_context():
+        context = build_thank_you_context(
+            fake_session_data, SUBMITTED_AT, SURVEY_TYPE_DEFAULT, SCHEMA
+        )
+
+        assert context["view_submitted_response"]["expired"] is False
+        assert context["view_submitted_response"]["url"] == "/submitted/view-response/"
+
+
+def test_view_submitted_response_expired_is_true_when_submitted_at_greater_than_expiry_time(
+    fake_session_data, app: Flask
+):
+    submitted_at = SUBMITTED_AT - timedelta(minutes=46)
+
+    with app.app_context():
+
+        context = build_thank_you_context(
+            fake_session_data, submitted_at, SURVEY_TYPE_DEFAULT, SCHEMA
+        )
+
+        assert context["view_submitted_response"]["expired"] is True
+
+
```

Comment:
These tests have been added between other tests, they should be moved to after the `test_default_survey_context_with_trad_as` method.